proposals/010-will-fund-for-food.md

```diff
@@ -0,0 +1,89 @@
+# Will Fund For Food Proposal
+## Liquidity Advertisements
+
+## Problem Description
+
+Sourcing liquidity on the lightning network is a common problem
+for any node looking to accept inbound payments.
+
+## Overview
+
+This proposal adds a TLV to the `node_announcement`,
```

Comment:
Is it worth splitting this part off into its own PR? Extensible gossip probably has uses beyond liquidity ads.